docs/core/testing/unit-testing-with-mstest.md

```diff
@@ -0,0 +1,215 @@
+CD into the `PrimeService.Tests` directory and create a new project using
+`dotnet new mstest`. This creates a test project
+that uses mstest as the test library. 
+
+The generated template configured the test runner
+in the PrimeServiceTests.csproj:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20170123-02" />
+  <PackageReference Include="MSTest.TestAdapter" Version="1.1.10-rc2" />
+  <PackageReference Include="MSTest.TestFramework" Version="1.0.8-rc2" />
+</ItemGroup>
+```
+
+The test project requires other packages to create and run unit tests.
+`dotnet new` added the MSTest SDK, the MSTest test framework and the MSTest runner. You need to add the PrimeService
+package as another dependency to the project. You can do that using the `dotnet`
+CLI:
+
+```
+dotnet add reference ../PrimeService/PrimeService.csproj
+```
+
+Or, you can directly edit the `PrimeService.Tests.csproj` file.
+Directly under the first
+`<ItemGroup>` node, add another `<ItemGroup>` node with a reference to 
+the library project:
+
+```xml
+  <ItemGroup>
+    <ProjectReference Include="..\PrimeService\PrimeService.csproj" />
+  </ItemGroup>
+```
+
+You can see the entire file in the
+[samples repository](https://github.com/dotnet/docs/blob/master/samples/core/getting-started/unit-testing-using-mstest/PrimeService.Tests/PrimeService.Tests.csproj) 
+on GitHub.
+
+After this initial structure is in place, you can write your first test.
+Once you verify that first unit test, everything is configured and should run smoothly
+as you add features and tests.
+
+## Creating the first test
+
+Before building the library or the tests, you need to run `dotnet restore`
+in both the `PrimeService` and `PrimeService.Tests` directories. This
+command restores all the necessary NuGet packages for each project.
+
+The TDD approach calls for writing one failing test, then making it pass,
+then repeating the process. So, let's write that one failing test. Remove
+`UnitTest1.cs` from the `PrimeService.Tests` directory, and create a new
+C# file named `PrimeService_IsPrimeShould.cs` with the following content:
+
+```cs
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Prime.Services;
+
+namespace Prime.UnitTests.Services
+{
+    [TestClass]
+    public class PrimeService_IsPrimeShould
+    {
+        private readonly PrimeService _primeService;
+        public PrimeService_IsPrimeShould()
+        {
+            _primeService = new PrimeService();
+        }
+
+        [TestMethod]
+        public void ReturnFalseGivenValueOf1()
+        {
+            var result = _primeService.IsPrime(1);
+
+            Assert.IsFalse(result, $"1 should not be prime");
+        }
+    }
+}
+```
+
+The `[TestClass]` attributes denotes a class that contains
+unit tests. The `[TestMethod]` attribute denotes a method as a single test. 
+
+Save this file, then run `dotnet build` to build the test project.
```

Comment:
You can do either. I wanted to just show both verbs at some point. Do you think that will cause confusion?